styles.css

```diff
@@ -110,8 +112,117 @@ a {
   font-size: 0.7rem;
 }
 
-.social_media_links {
+.social_media_links,
+.work ul {
   display: flex;
   gap: 1rem;
   margin-top: 1.2rem;
 }
+
+img {
+  height: 250px;
+  width: 100%;
+  margin-bottom: 0.7rem;
+}
+
+.work h2 {
+  font-weight: 700;
+  font-size: 2.2em;
+  color: #7d73fc;
+}
+
+.work_one,
+.work_two,
+.work_three,
+.work_four {
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+}
```

Comment:
- [ ] Kindly make sure to stick to the provided Figma design by using the correct fonts and please make sure to add margins in order to meet the design 😉. You can import the font from the [google font](https://fonts.google.com) site.
- [ ] Make sure to place your background images in the right position as shown in the Figma design.

   - ### Your design
  ![project](https://user-images.githubusercontent.com/87197412/185689785-612d274c-ecc1-4a25-a99a-9018888facd2.PNG)


   - ### Figma design
  ![real-work](https://user-images.githubusercontent.com/87197412/185689763-114cf8d7-414a-4f22-9927-57eff9615e00.PNG)

Reply:
Kindly you need to work with what the previous reviewer commented
_"Make sure to place your background images in the right position as shown in the Figma design."_